perfdash/config.go

```diff
@@ -384,10 +384,10 @@ func getProwConfig(configPaths []string) (Jobs, error) {
 	jobs := Jobs{}
 
 	for _, configPath := range configPaths {
-		fmt.Fprintf(os.Stderr, "Fetching config %s\n", configPath)
+		klog.Errorf("Fetching config %s\n", configPath)
 		// Perfdash supports only yamls.
 		if !strings.HasSuffix(configPath, ".yaml") {
-			fmt.Fprintf(os.Stderr, "%s is not an yaml file!\n", configPath)
+			klog.Errorf("%s is not an yaml file!\n", configPath)
```

Comment:
Also, this would be better as INFO or WARNING in my opinion (can happen for directories with OWNERS file and it is totally fine).